x-pack/plugins/reporting/server/lib/enqueue_job.test.ts

```diff
@@ -42,7 +39,7 @@ describe('Enqueue Job', () => {
       runTaskFnFactory: () => async () =>
         (({ runParamsTest: { test2: 'yes' } } as unknown) as TaskRunResult),
     });
-    mockReporting = await createMockReportingCore(mockConfig);
+    mockReporting = await createMockReportingCore(createMockConfigSchema());
```

Comment:
`createMockReportingCore` has to have a "raw" config passed, so the `syncConfig` in the ReportingCore constructor has working values. This requirement meant a lot of test mocks needed changes like the kind in this file.